onnx/defs/tensor/defs.cc

```diff
@@ -1358,35 +1358,75 @@ ONNX_OPERATOR_SET_SCHEMA(
         })
 );
 
-static const char* Resize_ver10_doc = R"DOC(
+static const char* Resize_ver11_doc = R"DOC(
 Resize the input tensor.
 Each dimension value of the output tensor is:
   output_dimension = floor(input_dimension * scale).
 )DOC";
 
 ONNX_OPERATOR_SET_SCHEMA(
     Resize,
-    10,
+    11,
     OpSchema()
         .Attr(
             "mode",
-            "Two interpolation modes: nearest (default), and linear (including bilinear, trilinear, etc)",
+            "Three interpolation modes: nearest (default), linear and cubic. "
+            "The \"linear\" mode includes linear interpolation for 1D tensor and N-D linear interpolation for N-D tensor (For example, trilinear interpolation for 2D tensor). "
```

Comment:
> "The \"linear\" mode includes linear interpolation for 1D tensor and N-D linear interpolation for N-D tensor (For example, trilinear interpolation for 2D tensor). "

should it be "_bilinear_ interpolation for 2D tensor" ?

Reply:
Thanks! It is fixed.